packular.py

```diff
@@ -171,7 +174,11 @@ def configure():
     opts = parse_options()
     defaults = dict(v.split('=') for v in opts.S or [])
     with open(opts.config_file) as config:
-        return read_config(config, defaults, opts.ignore_colon)
+        targets = read_config(config, defaults, opts.ignore_colon)
+    if opts.T:
+        return {k: targets[k] for k in targets if k == opts.T}
```

Comment:
maybe simplify to ``{opts.T: targets[opts.T]}`` - will also raise an exception if the target does not exist